### PR TITLE
New package r-spades-tools at 0.3.6

### PR DESCRIPTION
**Pull request by dorton21**

### CODE REVIEW

**Comment by dorton21** [comment]
depends on #21777

File: var/spack/repos/builtin/packages/r-spades-tools/package.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    depends_on('r-fastmatch@1.1-0:', type=('build', 'run'))
+    depends_on('r-fpcompare@0.2.1:', type=('build', 'run'))
+    depends_on('r-magrittr', type=('build', 'run'))
+    depends_on('r-quickplot', type=('build', 'run'))
+    depends_on('r-raster@2.5-8:', type=('build', 'run'))
+    depends_on('r-rcpp@0.12.12:', type=('build', 'run'))
+    depends_on('r-reproducible@0.2.0:', type=('build', 'run'))
+    depends_on('r-rgeos', type=('build', 'run'))
+    depends_on('r-sp@1.2-4:', type=('build', 'run'))
+    depends_on('r-randomfields', type=('build', 'run'))
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-spades-tools/package.py]: This is a suggested dependency so should be removed.

**Review by glennpj — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/r-spades-tools/package.py (hunk 1 above)
Comment (dorton21) [reply var/spack/repos/builtin/packages/r-spades-tools/package.py]: Are you referring to random fields or???

**Review by glennpj — APPROVED** [review]

**Comment by glennpj** [comment]
Close/open PR to trigger tests.

File: var/spack/repos/builtin/packages/r-spades-tools/package.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/SpaDES.tools"
+
+    maintainers = ['dorton21']
+
+    version('0.3.6', sha256='661f8ee792874e7447be78103775b63f18ec69e773a7b275dd977adb406dd3e5')
+
+    depends_on('r@3.5.0:', type=('build', 'run'))
+    depends_on('r-backports', type=('build', 'run'))
+    depends_on('r-checkmate@1.8.2:', type=('build', 'run'))
+    depends_on('r-circstats@0.2-4:', type=('build', 'run'))
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-spades-tools/package.py:29]:
```suggestion
    depends_on('r-circstat@0.2-4:', type=('build', 'run'))
```

**Review by glennpj — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/r-spades-tools/package.py (hunk 2 above)
Comment (glennpj) [reply var/spack/repos/builtin/packages/r-spades-tools/package.py:29]: Actually, this is correct, the `r-circstat` package needs to be renamed `r-circstats`.

Comment (glennpj) [inline var/spack/repos/builtin/packages/r-spades-tools/package.py:29]: This package needs a package for `r-circstats`. The `r-circstat` package in #21777 was supposed to be that package, but was misnamed. The best course is to rename that package with new PR.

**Review by glennpj — CHANGES_REQUESTED** [review]

**Comment by dorton21** [comment]
Check out #21966 for the fixed circstats

**Review by glennpj — APPROVED** [review]

**Pull request merged**